src/client/features/search/query-entity-info.js

```diff
@@ -144,6 +139,7 @@ const getUniprotInfo = ids => {
 
```

Comment:
Consider `let matchedDb = dbInfos[ db.type ];` - will this variable ever be used (non null)?

Reply:
It is repeat code.